src/test/fuzz/transaction.cpp

```diff
@@ -103,6 +105,5 @@ FUZZ_TARGET_INIT(transaction, initialize_transaction)
     (void)IsWitnessStandard(tx, coins_view_cache);
 
     UniValue u(UniValue::VOBJ);
-    TxToUniv(tx, /* hashBlock */ uint256::ZERO, u);
-    TxToUniv(tx, /* hashBlock */ uint256::ONE, u);
+    TxToUniv(tx, /* hashBlock= */ uint256{fuzzed_data_provider.ConsumeBool()}, /* entry= */ u, fuzzed_data_provider.ConsumeBool());
```

Comment:
not a fan of re-using the same buffer, thus nVersion for this bool.

`try { ds >> bool{}; }` might be nicer

Reply:
Makes sense, done 08543bf. I may have butchered this, so if you want to see any other changes here just lmk. Happy to fix up any nits bc I have a feeling there is a cleaner way to do this that I'm just not aware of